doc/integration.rst

```diff
@@ -103,10 +101,8 @@ Interfaces
 +-------------------------+-------------------------+-----+----------------------------------------+
 | ``test_en_i``           | 1                       | in  | Test input, enables clock              |
 +-------------------------+-------------------------+-----+----------------------------------------+
-| ``core_id_i``           | 4                       | in  | Core ID, usually static, can be read   |
+| ``hart_id_i``           | 32                      | in  | hart ID, usually static, can be read   |
```

Comment:
tiny nit: Everything in this column starts with a capital letter, so you might want to do the same here.